### PR TITLE
stat missing in the function description

### PR DESCRIPTION
**Pull request by NicolasClairis**

### CODE REVIEW

File: stats&plots/RFT_GLM_contrast.m [hunk 1]
```diff
@@ -25,6 +25,7 @@
 %   - u: set-inducing threshold (for cluster inference)
 %   - verbose: flag for displaying results (default is 0)
 % OUT:
+%   - stat:
```
Comment (lionel-rigoux) [inline stats&plots/RFT_GLM_contrast.m]: There should be a description of what `stat`is.

File: stats&plots/RFT_GLM_contrast.m [hunk 2]
```diff
@@ -1,6 +1,6 @@
 function [stat,out] = RFT_GLM_contrast(X,y,c,type,u,verbose)
 % applies RFT to GLM-based contrast inference
-% function [out] = RFT_GLM_contrast(X,y,c,type,verbose)
+% function [stat,out] = RFT_GLM_contrast(X,y,c,type,verbose)
```
Comment (lionel-rigoux) [inline stats&plots/RFT_GLM_contrast.m]: `u` input is also missing un the doc

**Comment by lionel-rigoux** [comment]
Hey,

Thanks for correcting the documentation!
I spotted some additional typos. I don't know this function, could you update your patch before I merge it to the master?

Cheers

File: stats&plots/RFT_GLM_contrast.m (hunk 1 above)
Comment (NicolasClairis) [reply stats&plots/RFT_GLM_contrast.m]: @jdaunize ?

**Comment by lionel-rigoux** [comment]
Is this still active? It shouldn’t be too long to add a couple of lines of documentation ;)

**Comment by NicolasClairis** [comment]
Si mais je t'avais répondu sur ton adresse gmail qui ne semble plus fonctionner(?) C'est la fonction de Jean donc je sais pas trop ce que "stats" sort comme valeurs, je crois que c'est les t.values, Jules tu sais?

Nicolas

**Comment by lionel-rigoux** [comment]
Note that answering to an email initiated by a github issue/pull request is automatically added to the issue thread and forwarded to all the contributors of the repo.
You didn’t send your last email to my personal gmail address, you contributed to the github discussion (as you can see here: https://github.com/MBB-team/VBA-toolbox/pull/47 <https://github.com/MBB-team/VBA-toolbox/pull/47>) and that was broadcasted to other VBA contributors including Jean.
Given his (understandably) laconic and surprised response, I am not sure he will take care of this. And pointing fingers won’t make things go forward...

I suggest you talk to each others and decide who’s pushing a complete patch. If this pull request is not updated in the upcoming week, I will just close the pull request and flag the missing doc in a separate issue.

File: stats&plots/RFT_GLM_contrast.m (hunk 1 above)
Comment (NicolasClairis) [reply stats&plots/RFT_GLM_contrast.m]: stat: vector containing the t- or F-values (depending on the test type selected in the input) corresponding to each point in Y

**Pull request merged**